### PR TITLE
Create a proper readme file

### LINKED ISSUE #2: Create a proper Readme file.

### PR DESCRIPTION
**Pull request by RVany**
#2 
I used the README of the existing repos as reference.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,24 +14,17 @@ Short summary on functionality and used technologies.
...
-| folder name | Author details (name, company, twitter alias with link) |
+| custom-welcome-banner | [gcxchange-gcechange](https://github.com/gcxchange-gcechange) |
 
 ## Version history
 
 | Version | Date             | Comments        |
 | ------- | ---------------- | --------------- |
-| 1.1     | March 10, 2021   | Update comment  |
-| 1.0     | January 29, 2021 | Initial release |
+| 1.0     | June, 2022 | Initial release |
```
Comment (Steph4104) [inline README.md]: small fix. Date will be July

File: README.md [hunk 2]
```diff
@@ -1,10 +1,9 @@
 # custom-welcome-banner
 
 ## Summary
+Custom welcome banner is a small web part developed using sharepoint react frame work. It is available in both English ang French languages. Language can be selected throug the property pane.
```
Comment (Steph4104) [inline README.md]: Small typo: Framework

File: README.md [hunk 3]
```diff
@@ -41,31 +33,22 @@ Short summary on functionality and used technologies.
...
-- topic 1
-- topic 2
-- topic 3
-
-> Notice that better pictures and documentation will increase the sample usage and the value you are providing for others. Thanks for your submissions advance.
-
-> Share your web part with others through Microsoft 365 Patterns and Practices program to get visibility and exposure. More details on the community, open-source projects and other activities from http://aka.ms/m365pnp.
+This Web Part illustrates the following concepts on top of the SharePoint Framework:
+* Using the SPFx context to gather the current user's display name.
+* Language selection (English/French) through the property pane.
```
Comment (Steph4104) [inline README.md]: You also have the account option. Maybe give more detail around it. Like it match the user profile language

**Review by Steph4104 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RVany) [reply README.md]: Fixed it.

File: README.md (hunk 2 above)
Comment (RVany) [reply README.md]: Fixed it.

File: README.md (hunk 3 above)
Comment (RVany) [reply README.md]: Updated.

**Review by Steph4104 — APPROVED** [review]

**Comment by RVany** [comment]
@Steph4104 I think we need to modify the read me file, according to the updated web part. Now web part is not limited to English , French  and Account. It can be used for any languages. Preferred languages field is removed from property pane. @piet0024 is requested to update the validation for URL field, so I updated yesterday and waiting for the review. Please take a look #6 when you have time.

**Comment by Steph4104** [comment]
@RVany Yes, you are right. You can update the readme from here or from the branch https://github.com/gcxchange-gcechange/custom-welcome-banner/issues/6.

**Comment by RVany** [comment]
> Yes, you are right. You can update the readme from here or from the branch #6.

@Steph4104 Since you approved this PR.  I merge this to main and will update the readme file #6. Thank you!.

**Pull request merged**